.github/workflows/playwright.yml

```diff
@@ -0,0 +1,33 @@
+name: Playwright Tests
+on:
+  pull_request:
+    branches: [ main, master ]
+  workflow_dispatch:
+
+jobs:
+  test:
+    timeout-minutes: 10
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+    - uses: actions/setup-node@v4
+      with:
+        node-version: lts/*
+    - name: Install dependencies
+      run: npm ci
+    - name: Install Playwright Browsers
+      run: npx playwright install chromium --with-deps
+    - name: Run Playwright tests
+      run: npx playwright test
+    - uses: actions/upload-artifact@v4
+      if: ${{ !cancelled() }}
+      with:
+        name: html-report
+        path: playwright-report/
+        retention-days: 5
+    - uses: actions/upload-artifact@v4
+      if: ${{ !cancelled() }}
+      with:
+        name: traces
+        path: test-results/
+        retention-days: 5
```

Comment:
If you have any questions about this, I'm an open book. But also, asking any AI chat will probably give you a good explanation of anything that looks unfamiliar here 😁